### PR TITLE
Re-introduce `strip-line` as `DEVELOPMENT_TEAM` is being populated again

### PR DESCRIPTION
**Pull request by domesticmouse**
## Pre-launch Checklist

- [x] I read the [Effective Dart: Style] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Effective Dart: Style]: https://dart.dev/guides/language/effective-dart/style
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: tooling/codelab_rebuild/lib/src/rebuild_blueprint.dart [hunk 1]
```diff
@@ -233,6 +233,15 @@ Future<void> _buildBlueprintStep(Directory cwd, BlueprintStep step) async {
     return;
   }
 
+  final stripLinesContaining = step.stripLinesContaining;
+  if (stripLinesContaining != null) {
+    final target = File(p.join(cwd.path, step.path));
+    var lines = target.readAsLinesSync();
```
Comment (parlough) [inline tooling/codelab_rebuild/lib/src/rebuild_blueprint.dart:239]: Not important, but seems like the rest of your file uses `final`:

```suggestion
    final lines = target.readAsLinesSync();
```

File: tooling/codelab_rebuild/lib/src/rebuild_blueprint.dart [hunk 2]
```diff
@@ -233,6 +233,15 @@ Future<void> _buildBlueprintStep(Directory cwd, BlueprintStep step) async {
...
+    lines.removeWhere((line) => line.contains(stripLinesContaining));
+    target.writeAsStringSync(lines.join('\n'));
```
Comment (parlough) [inline tooling/codelab_rebuild/lib/src/rebuild_blueprint.dart:241]: Do consider adding an extra new line at the end to write out, so the no new line at EOF symbols that GitHub shows don't bother us too much :)

**Review by parlough — APPROVED** [review]
<sub>_Déjà vu_</sub>

Looks good to me! Two optional things to consider :) 

OHH! Also, happy holidays Brett! I hope you have a great rest of the year :D

Thanks for all of the support you've continued to provide for me the past few years.

File: tooling/codelab_rebuild/lib/src/rebuild_blueprint.dart (hunk 2 above)
Comment (domesticmouse) [reply tooling/codelab_rebuild/lib/src/rebuild_blueprint.dart]: Done

**Pull request merged**